docs/events.yaml

```diff
@@ -64,7 +64,7 @@ x-events:
     cluster_id: UUID
 
 - name: cluster_validation_failed
-  message: "Cluster validation '{validation_id}' that used to succeed is now failing"
+  message: "Cluster validation '{validation_id}' failed"
```

Comment:
Mind [the signatures](https://github.com/openshift-assisted/assisted-installer-deployment/blob/cef68218e58164b13b6af26cf14a037af70ad95f/tools/add_triage_signature.py#L1552-L1553)

Reply:
I suspected that might be the case.
I'll restore the original logging in case the validation used to pass

Reply:
Unsure if you suggest to update the signature or add a new event for failing validation (that didn't use to succeed)

Reply:
Assuming the former, reopen if you think it should be the latter

Reply:
Not suggesting anything, just make sure they're compatible / still making sense when you're done

Reply:
Eventually I decided to keep the old event message in case the status changed from success to failure

Reply:
You wrote that you'll keep the old message, but it seems that you didn't.